docs/reference/builder.md

```diff
@@ -908,6 +909,7 @@ Each `<src>` may contain wildcards and matching will be done using Go's
 
     COPY hom* /mydir/        # adds all files starting with "hom"
     COPY hom?.txt /mydir/    # ? is replaced with any single character, e.g., "home.txt"
+    COPY arr[[]0].txt /mydir/    # file name will be "arr[0].txt"
```

Comment:
Oh, looks like you forgot to remove this one (it's now moved to below, right?). May have missed that 😅